internal/params/config.go

```diff
@@ -98,6 +99,7 @@ var (
 		SHA3Epoch:                  big.NewInt(74570),
 		HIP6And8Epoch:              big.NewInt(74570),
 		StakingPrecompileEpoch:     big.NewInt(75175),
+		ChainIdFixEpoch:            big.NewInt(75175),
```

Comment:
Set it to be future epoch. Forking can be not retrospective.

Reply:
I changed the epoch to 75685.